core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedSchedulerBackend.scala

```diff
@@ -557,6 +557,10 @@ class CoarseGrainedSchedulerBackend(scheduler: TaskSchedulerImpl, val rpcEnv: Rp
     executorDataMap.keySet.toSeq
   }
 
+  def getExecutorsWithRegistrationTs(): Map[String, Long] = synchronized {
+    executorDataMap.mapValues(v => v.registrationTs).toMap
```

Comment:
nit: `_.registrationTs`

Reply:
Hi, @jaceklaskowski . In a backport PR, we had better keep the consistency between branches. We can do that later from the `master` branch. 

Please don't change this part, @attilapiros .